src/ngraph_cluster_manager.cc

```diff
@@ -39,6 +39,14 @@ GraphDef* NGraphClusterManager::GetClusterGraph(int idx) {
 
 void NGraphClusterManager::EvictAllClusters() { s_cluster_graphs.clear(); }
 
+// TODO: Replace vector with map
```

Comment:
The PR to replace map with vector is hitting some issues: https://github.com/tensorflow/ngraph-bridge/pull/34
so for now still using vector in `NGraphClusterManager`